pages/index.js

```diff
@@ -57,75 +64,95 @@ export default function HomePage() {
     setMode("select");
   }
 
+  function handleIncreaseStats(attribute, steps) {
+    setSelectedPet((prevPet) => {
+      const newValue = Math.min(prevPet[attribute] + steps, 100);
+      console.log(newValue);
+      console.log(prevPet);
+      return {
+        ...prevPet,
+        [attribute]: newValue,
+      };
+    });
+  }
+
   return (
     <>
       {mode === "select" && (
         <>
           <StyledGrid>
-          <StyledHeading $variant="select">Select a Survival Pet</StyledHeading>
-          <StyledSection >
-            <PetList onSelectPet={handleSelectPet} selectedPet={selectedPet} />
-          </StyledSection>
-          
-          <PetForm selectedPet={selectedPet} onSubmit={handleSubmit} />
+            <StyledHeading $variant="select">
+              Select a Survival Pet
+            </StyledHeading>
+            <StyledSection>
+              <PetList
+                onSelectPet={handleSelectPet}
+                selectedPet={selectedPet}
+              />
+            </StyledSection>
+            <PetForm selectedPet={selectedPet} onSubmit={handleSubmit} />
           </StyledGrid>
         </>
       )}
 
       {mode === "livingroom" && (
         <>
           <StyledGrid>
-          <StyledHeading $variant="livingroom">Living Room</StyledHeading>
-          {!isDead ? (
-            <StyledSection>
-              <CurrentPet selectedPet={selectedPet} />
-            </StyledSection>
-          ) : (
-            <StyledSection>
-              <TombstoneButton
-                selectedPet={selectedPet}
-                onMode={handleMode}
-                onDeletePet={handleDeletePet}
-              />
-            </StyledSection>
-          )}
-          <CurrentPetStats
-            isDead={isDead}
-            selectedPet={selectedPet}
-            onMode={handleMode}
-          />
+            <StyledHeading $variant="livingroom">Living Room</StyledHeading>
+            {!isDead ? (
+              <StyledTest>
+                <div></div>
```

Comment:
What is this empty `div` needed for?

Reply:
It is needed for our grid layout. we have a 20% 60% 20% structure, and the div has to be there for the moment.